modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -5,167 +5,28 @@
 [id="dr-scenario-3-recovering-expired-certs_{context}"]
 = Recovering from expired control plane certificates
 
-Follow this procedure to recover from a situation where your control plane certificates have expired.
+OpenShift can now automatically recover control plane certificates, replacing the previous manual steps.
 
-.Prerequisites
+When recovering kubelet certificates you need to manually approve the pending `node-bootstrapper` certificates signing requests (CSRs).
```

Comment:
* Comma after kubelet certificates
* you need to -> you must
* certificates signing requests -> certificate signing requests

Reply:
done, thx